src/amo/css/AddonDetail.scss

```diff
@@ -16,6 +16,7 @@ $page-margin: 20px 10px;
 .AddonDetail-title-heading {
   font-size: $font-size-l;
   margin-bottom: 10px;
+  word-break: break-all;
```

Comment:
It's better to use `word-wrap: break-word;` we had problems with this in the past see https://github.com/mozilla/addons-frontend/issues/2319